Neos.Neos/Resources/Private/Styles/Modules/Administration/_Users.scss

```
@@ -0,0 +1,13 @@
+.users-table {
+	thead {
+		tr {
+			th {
+				a {
+					&:hover {
+						color: #{$blue}
+					}
+				}
+			}
+		}
+	}
+}
```

i think that is a bit to much nesting 😂 
```suggestion
.users-table thead th a:hover {
  color: #{$blue}
}
```